qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -156,52 +156,83 @@ def to_printer(self, printer, callback=None):
         self._widget.page().print(printer, callback)
 
 
+class _WebEngineSearchWrapHandler():
```

Comment:
nitpick:

```suggestion
class _WebEngineSearchWrapHandler:
```

(the parentheses are optional when not inheriting from anything, and we leave them off them in other places)